src/Apps/Settings/Routes/Insights/InsightsRoute.tsx

```diff
@@ -23,10 +25,26 @@ const InsightsRoute: React.FC<InsightsRouteProps> = ({ me }) => {
       {isInsightsEnabled && (
         <>
           <InsightsHeader />
+
           <Join separator={<Spacer my={[4, 6]} />}>
             {/* @ts-ignore RELAY_UPGRADE 13 */}
             <InsightsOverviewFragmentContainer info={me?.myCollectionInfo!} />
 
+            <Media greaterThanOrEqual="sm">
+              <InsightsCareerHighlightRailFragmentContainer
+                // @ts-ignore RELAY_UPGRADE 13
+                me={me}
+                showProgress={true}
+              />
+            </Media>
+            <Media lessThan="sm">
+              <InsightsCareerHighlightRailFragmentContainer
+                //@ts-ignore RELAY_UPGRADE 13
+                me={me}
+                showProgress={false}
+              />
+            </Media>
```

Comment:
Used `<Media>` here to show or hide the progress bar using Server Side Rendering instead of using Client side with `__internal__useMatchMedia`